src/Idris/Error.idr

```diff
@@ -257,19 +257,19 @@ perror (BadDataConType fc n fam)
 perror (NotCovering fc n IsCovering)
     = pure $ errorDesc (reflow "Internal error" <++> parens (reflow "Coverage of" <++> code (pretty n)))
 perror (NotCovering fc n (MissingCases cs))
-    = pure $ errorDesc (code (pretty !(prettyName n)) <++> reflow "is not covering.")
+    = pure $ errorDesc (code (pretty !(toFullNames n)) <++> reflow "is not covering.")
         <+> line <+> !(ploc fc) <+> line
         <+> reflow "Missing cases" <+> colon <+> line
         <+> indent 4 (vsep !(traverse (pshow []) cs)) <+> line
 perror (NotCovering fc n (NonCoveringCall ns))
-    = pure $ errorDesc (pretty !(prettyName n) <++> reflow "is not covering.")
+    = pure $ errorDesc (pretty !(toFullNames n) <++> reflow "is not covering.")
         <+> line <+> !(ploc fc) <+> line
         <+> reflow "Calls non covering function" <+>
         case ns of
              [fn] => space <+> pretty fn
              _ => pretty 's' <+> colon <++> concatWith (surround (comma <+> space)) (pretty <$> ns)
 perror (NotTotal fc n r)
-    = pure $ errorDesc (code (pretty !(prettyName n)) <++> reflow "is not total," <++> pretty r)
+    = pure $ errorDesc (code (pretty !(toFullNames n)) <++> reflow "is not total," <++> pretty r)
```

Comment:
The reason probably needs a `toFullNames` too as that was one of the $ names reported on the discord.